testnet/stacks-node/src/nakamoto_node/relayer.rs

```diff
@@ -1514,24 +1721,26 @@ impl RelayerThread {
         ))
     }
 
-    /// Try to start up a tenure-extend, after a delay has passed.
-    /// We would do this if we were the miner of the ongoing tenure, but did not win the last
-    /// sortition, and the winning miner never produced a block.
+    /// Try to start up a tenure-extend.
+    /// Only do this if the miner won the last-ever sortition but the burn view has changed.
```

Comment:
Should this say "the last valid sortition"?

Reply:
yes